### DISCUSSION TITLE
Issue with dropdowns in p-table, when opened in Thomson Reuters Financial system 3E

### DISCUSSION THREAD

**Discussion by MarcAZ91** [primefaces/community#675] [PrimeNG]
Dear support,

I am facing an issue with a application I have built that will be opened inside Thomson Reuters 3E financial system.
When I try pressing on any of the generated dropdowns for filtering in p-table or page size dropdown the entire 3E sites shifts to the left

Any ideas :D 
?

Thank you in advance

**Comment by cetincakiroglu** [comment]
Hi,

I don't know about `Thomson Reuters 3E financial system` but we haven't received such an issue before. Could you please share a stackblitz example or code of the related sections to identify the issue clearly?

**Comment by cetincakiroglu** [comment]
Closing the topic due to the inactivity.
